File: backend/app/api/tasks.py
from datetime import datetime
from fastapi import APIRouter
from app.schemas.task import Task

router = APIRouter()


@router.get("/", response_model=list[Task])
def get_tasks():
  now = datetime.now().strftime("%Y-%m-%d")

  return [
    Task(
      id="1",
      title="Сделать MVP",
      description="Первая версия приложения",
      status="todo",
      createdAt=now,
      updatedAt=now,
    ),
    Task(
      id="2",
      title="Сделать графики",
      status="inProgress",
      createdAt=now,
      updatedAt=now,
    ),
  ]

File: client/src/mocks/tasks.ts
import type {Task} from "@/app/types/types.ts";

const tasks: Task[] = [
  { id: '1', title: 'Сделать MVP', description: 'Сделать эту задачу в свое время', status: 'todo', createdAt: '2026-01-02', updatedAt: '2026-01-02' },
  { id: '2', title: 'Сделать графики', description: 'Сделать эту задачу в свое время', status: 'inProgress', createdAt: '2026-01-02', updatedAt: '2026-01-02' },
  { id: '3', title: 'Протестировать Skeleton', description: 'Сделать эту задачу в свое время', status: 'done', createdAt: '2026-01-02', updatedAt: '2026-01-02' },
]

export function fetchTasksMock(): Promise<Task[]> {
  return new Promise(resolve => {
    setTimeout(() => resolve(tasks), 2000)
  })
}